### PR TITLE
demystifying-web/rediscovering-the-browser: Rework session content

### PR DESCRIPTION
**Pull request by Ryx076**
Transitioned the 'rediscovering-the-browser' directory to meet the requirements of the OpenEdu Methodology.

# Prerequisite Checklist
- [ ] Read the [contribution guidelines](https://github.com/open-education-hub/ccas-internal/blob/main/CONTRIBUTING.md#pull-requests) regarding submitting new changes to the project;
- [ ] Tested your changes against relevant architectures and platforms;
- [ ] Updated relevant documentation (if needed).

### CODE REVIEW

File: chapters/demystifying-web/rediscovering-the-browser/guides/readme/sol/solution.sh [hunk 1]
```diff
@@ -14,6 +14,6 @@ fi
 # Readme
 echo "Start exploit for Readme"
 remote=$remote'/readme/'
-flag=$(curl -s $remote | grep -o "SSS_CTF{.*}")
+flag=$(curl -s "$remote" | grep -o "SSS_CTF{.*}")
 echo "Flag is $flag"
 echo "----------------------------"
```
Comment (teodutu) [inline chapters/demystifying-web/rediscovering-the-browser/guides/readme/sol/solution.sh]:
```suggestion
echo "----------------------------"

```

**Review by teodutu — CHANGES_REQUESTED** [review]
- Rename the `sol/` folders to `solution/`
- At the end of each task’s README.md, add a sentence like, "If you're having difficulties solving this exercise, go through [this](link to the relevant section in reading/) reading material."

Other than this, the content looks good.

**Comment by Ryx076** [comment]
> * At the end of each task’s README.md, add a sentence like, "If you're having difficulties solving this exercise, go through [this](link to the relevant section in reading/) reading material."

What tasks are you referring to? @teodutu

**Review by teodutu — APPROVED** [review]
:ok_hand:

**Comment by teodutu** [comment]
> > * At the end of each task’s README.md, add a sentence like, "If you're having difficulties solving this exercise, go through [this](link to the relevant section in reading/) reading material."
> 
> What tasks are you referring to? @teodutu

Nevermind, I didn't see that they had that line. GG.

**Pull request merged**